.github/workflows/build.yml

```diff
@@ -140,103 +140,122 @@ jobs:
         with:
           fetch-depth: 0
       - name: Build
-        run: docker build --target release -t $DOCKER_IMAGE_ID .
+        run: |
+          docker buildx create \
+            --name multibuilder \
+            --platform linux/amd64,linux/arm64 \
+            --bootstrap --use
+          docker buildx build \
+            --platform linux/amd64,linux/arm64 \
+            -t $DOCKER_IMAGE_ID .
```

Comment:
I am not a docker person and definitely not a buildx person, but the previous attempt at this also had changse to the docker file - https://github.com/grafana/k6/pull/3015

Are they not needed now? Were they never needed?

Reply:
They are the support for [cross-compilation](https://docs.docker.com/build/building/multi-platform/#cross-compilation), if I'm understanding buildx in the right way without using it we are instead emulating the platform. It has the cons of being slower. I would prefer if we could keep the PR separated.

>Using an emulator this way is very easy. We don’t need to modify our Dockerfile at all and can build for multiple platforms automatically. But it doesn’t come without downsides. The binaries running this way need to constantly convert their instructions between architectures and therefore don’t run with native speed. Occasionally you might also find a case that [triggers a bug](https://github.com/docker/buildx/issues?q=is%3Aopen+is%3Aissue+label%3Aqemu) in the emulation layer.

So, I'm for merging this PR and check that it builds fine then I will open a new PR specifically for the dockerfile and enabling the cross-compilation.